lib/engine/ability/destination.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
```

Comment:
i wonder if we don't even need the destination ability because we already have the assign_hexes ability

what do you think?